internal/testhelpers/e2e/spawn.go

```diff
@@ -74,7 +74,7 @@ func (s *SpawnedCmd) ExpectInput(opts ...termtest.SetExpectOpt) error {
 	expect := `expect'input from posix shell`
 	if cmdName != "bash" && shellName != "bash" && runtime.GOOS == "windows" {
 		if strings.Contains(cmdName, "powershell") || strings.Contains(shellName, "powershell") {
-			send = "echo \"`<expect input from powershell`>\""
```

Comment:
Are you sure this works? IIRC the backtick is an escape on powershell.

Reply:
You were right, I was running into a failure with this because it was defaulting back to `cmd.exe` and not powershell.

Reply:
Never mind, I was right. Here's the output on powershell

```
[ActiveState-CLI/small-python] C:\Users\runneradmin\AppData\Local\Temp\2179854665\work\small-python>echo "`<expect input from powershell`>" 
"`<expect input from powershell`>"    
```

And the expect

```
'Expected: "<expect input from powershell>": after 40s: timeout':
              'after 40s: timeout': timeout
```

Reply:
My guess is it depends on which shell you started from. So your new tests may want this variant but older tests will want the escaping? Worth perhaps looking for other places where we run ExpectInput and running those tests on Windows before merging your PR.

Reply:
All of the integration tests that start powershell are also critical. Only one of them expects input which is the one that flagged this change.